pkg/e2e/osd/daemonsets.go

```diff
@@ -0,0 +1,84 @@
+package osd
+
+import (
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func makeDaemonSet(name, sa string, nodeLabels map[string]string) appsv1.DaemonSet {
+	matchLabels := make(map[string]string)
+	matchLabels["name"] = name
+	ds := appsv1.DaemonSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: appsv1.DaemonSetSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: matchLabels,
+			},
+			Template: v1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:   name,
+					Labels: matchLabels,
+				},
+				Spec: v1.PodSpec{
+					NodeSelector:       nodeLabels,
+					ServiceAccountName: sa,
+					Containers: []v1.Container{
+						{
+							Name:  "test",
+							Image: "registry.access.redhat.com/ubi8/ubi-minimal",
+						},
+					},
+				},
+			},
+		},
+	}
+
+	return ds
+}
+
+var _ = ginkgo.Describe("[Suite: service-definition] [OSD] DaemonSets", func() {
+	ginkgo.Context("DaemonSets are not allowed", func() {
```

Comment:
Does this context make sense now? I'm assuming, since this is testing for positive existence of DaemonSets, that this isn't the intention any more.